user_guide_src/source/installation/upgrade_430.rst

```diff
@@ -156,6 +156,7 @@ Others
 Breaking Enhancements
 *********************
 
+- Since void element in ``html_helper`` and ``form_helper`` has been changed to html5 valid, your application with XHTML cant be valid.
```

Comment:
```suggestion
- Since void HTML element (e.g. ``<input>``) in ``html_helper``, ``form_helper`` or common functions has been changed to HTML5 valid by default. If you need to be compatible with XHTML, set the ``$html5`` property in **app/Config/DocTypes.php** to ``false``.
```

Reply:
Done